### PR TITLE
refact "cscli config"

### PR DESCRIPTION
**Pull request by mmetc**

### CODE REVIEW

File: cmd/crowdsec-cli/config_restore.go [hunk 1]
```diff
@@ -308,11 +294,19 @@ func NewConfigRestoreCmd() *cobra.Command {
...
+			}
+
+			return nil
+		},
 	}
 
-	flags := cmdConfigRestore.Flags()
-	flags.BoolP("old-backup", "", false, "To use when you are upgrading crowdsec v0.X to v1.X and you need to restore backup from v0.X")
+	flags := cmd.Flags()
+	flags.BoolVarP(&oldBackup, "old-backup", "", false, "To use when you are upgrading crowdsec v0.X to v1.X and you need to restore backup from v0.X")
```
Comment (LaurenceJJones) [inline cmd/crowdsec-cli/config_restore.go:309]: This being remove in this PR or next?

Comment (mmetc) [reply cmd/crowdsec-cli/config_restore.go:309]: Yes I'll remove it in a next commit

Comment (mmetc) [reply cmd/crowdsec-cli/config_restore.go]: done

**Review by sabban — APPROVED** [review]
lgtm

**Pull request merged**